tests/bench/oolong_common.py

```
@@ -0,0 +1,533 @@
+            gold_num = float(gold_answer)
+            return 0.75 ** abs(gold_num - model_num)
+        except (ValueError, TypeError):
+            return 0.0
+
+    # Date matching
+    if "DATE" in answer_type.upper():
+        try:
+            model_date = dateutil.parser.parse(model_answer)
+            if isinstance(gold_answer, datetime):
+                return 1.0 if model_date.date() == gold_answer.date() else 0.0
+            return 0.0
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)